lib/api/helpers.js

```diff
@@ -1,5 +1,5 @@
 import url from "url";
-import { slugify } from "transliteration";
+import "transliteration/lib/browser/transliteration.js";
```

Comment:
None of that is working. It looks like this has fixed the problem on the clients-side but broken it on the server-side.

I'm getting this error on the server-side whenever I start and it's also what is failing the tests
```
  TypeError: _slugify is not a function
      at _getSlug (lib/api/helpers.js:86:27)
      at Object.autoValue (lib/collections/schemas/products.js:467:9)
      at Object.runAV (packages/aldeed_simple-schema.js:1282:26)
      at packages/aldeed_simple-schema.js:1383:13
      at Array.forEach (native)
      at Function._.each._.forEach (packages/underscore.js:139:11)
      at packages/aldeed_simple-schema.js:1382:7
```

adding in any import for the server causes the client-side to fail again. I am convinced that https://github.com/andyhu/transliteration/issues/38 is related. We might want to pull in https://github.com/andyhu/transliteration/pull/44 and see if that fixes it.